bc_obps/registration/tests/utils/bakers.py

```diff
@@ -54,7 +54,17 @@ def operator_baker():
     )
 
 
-def operation_baker():
+def operation_baker(operator_id=None):
+    if operator_id:
+        return baker.make(
+            Operation,
+            operator=operator_baker(),
```

Comment:
This `operator` assignment is redundant since we are passing the `operator_id` on line 65.